recipes/conda-standalone/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+CONDA_VERSION:
+  - 4.7.12
+python:
+  - 3.7
```

Comment:
Is it necessary to pin these here?

Reply:
I followed https://github.com/AnacondaRecipes/conda-standalone-feedstock/tree/master/recipe

Should I remove this file?